### ISSUE TITLE
Add option to show overmap tiles in "LOOT"

### ISSUE THREAD

**Issue by Faalagorn** [nornagon/cdda-guide#66]
Similar to how items and monsters are shown, it would be nice if there would be an option to show graphics instead of ASCII in LOOT section of the how to obtain an item.

**Comment by nornagon** [comment]
Ah, can the overmap be graphical now? I thought that was still a mod-only thing.

**Comment by Faalagorn** [comment]
> Ah, can the overmap be graphical now? I thought that was still a mod-only thing.

Nah, it's there in the master for a while already, since July I think? https://github.com/CleverRaven/Cataclysm-DDA/pull/43203#issuecomment-873525067

**Comment by nornagon** [comment]
Ignore the incorrect symbols in this screenshot, but this shows one challenge here: some overmaps are huge, and while the ASCII version fits in a relatively small space, the tiled version does not.

<img width="850" alt="Screenshot 2023-10-29 at 10 32 46 PM" src="https://github.com/nornagon/cdda-guide/assets/172800/970b4241-02fe-4267-b797-98b1d8349886">
